### PR TITLE
added stealth worker image

### PR DESCRIPTION
**Pull request by shicholas**
[Stealth Worker](https://www.stealthworker.com) is using Ember w/ JSON API. We'd appreciate being listed on your user page.

Please let me know if there is anything besides our logo we need.

### CODE REVIEW

**Comment by shicholas** [comment]
and added data.users after looking at #2567

File: data/users.yml [hunk 1]
```diff
@@ -815,6 +815,10 @@
   image: airhelp.png
   alt: "AirHelp"
   use: "AirHelp web app is built with Ember"
+- url: https://www.stealthworker.png
```
Comment (bmac) [inline data/users.yml]: Looks like this url has the wrong top level domain. (`.png` vs `.com`).

Comment (shicholas) [inline data/users.yml]: gah, brainfart.

Comment (shicholas) [inline data/users.yml]: haha fixed, thanks for pointing this out!

**Comment by wifelette** [comment]
Needs a rebase, and email hasn't been received. Should be good to go once that happens.

**Comment by shicholas** [comment]
Sorry @wifelette where do I send an email? I was just looking on this projects readme

**Comment by wifelette** [comment]
Instructions are in the Yaml file :)

**Comment by shicholas** [comment]
rebased and email sent, thanks!

**Pull request merged**